websites/H/Home Assistant/presence.ts

```diff
@@ -0,0 +1,131 @@
+const presence = new Presence({
+    clientId: "812627990382772224"
+}), browsingStamp = Math.floor(Date.now() / 1000);
+  
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "home_assistant_logo" //banner
+  }; 
+  presenceData.startTimestamp = browsingStamp;
```

Comment:
just, add startTimestamp to presenceData object?

Reply:
What do you mean? We have to insert startTimestamp into const presenceData?